src/nodes/listen.js

```diff
@@ -74,7 +74,7 @@ module.exports = function(RED) {
         };
       }
       if (/^\d+$/.test(config.timeout)) obj.timeout = parseInt(config.timeout);
-      if (/^\d+$/.test(config.maxlength)) obj.maxLength = parseInt(config.maxLength);
+      if (/^\d+$/.test(config.maxlength)) obj.maxLength = parseInt(config.maxlength);
```

Comment:
I'm confused about this because `maxLength` is the valid property name, per [verb specs](https://github.com/jambonz/verb-specifications/blob/cb8b51da4a1cf67a19a1675939eb257b94cdb87c/specs.json#L87)

Reply:
Yes, but the problem is elsewhere. The value of `config.maxlength` is being checked, but after that, `parseInt(config.maxLength)` is assigned to `obj.maxLength`, where the 'L' is capitalized. This key does not exist in config [here](https://github.com/jambonz/node-red-contrib-jambonz/blob/main/src/nodes/listen.html#L24). The value of `config.maxlength` is assigned to obj, which matches the specification you sent. Am I wrong?